examples/reagent/src/cljs/todomvc/session.cljs

```diff
@@ -0,0 +1,12 @@
+(ns todomvc.session
+  (:require [reagent.core :as reagent]
+            [alandipert.storage-atom :refer [local-storage]]))
```

Comment:
It's a little unfortunate that the library doesn't support namespacing at all. We usually require apps to have a special key in localStorage. It seems like you'd have to implement your own StorageBackend with this. The problem is, that if we don't do it this time, the next example that tries to make use of the library here will break. Not quite sure what the best course of action would be.